src/components/modals/Send/fields/RecipientField.js

```diff
@@ -39,12 +40,16 @@ class RecipientField<Transaction> extends Component<Props<Transaction>, { isVali
   async resync() {
     const { account, bridge, transaction } = this.props
     const syncId = ++this.syncId
-    const isValid = await bridge.isRecipientValid(
-      account.currency,
-      bridge.getTransactionRecipient(account, transaction),
-    )
-    if (syncId !== this.syncId) return
-    this.setState({ isValid })
+    try {
+      const isValid = await bridge.isRecipientValid(
+        account.currency,
+        bridge.getTransactionRecipient(account, transaction),
+      )
+      if (syncId !== this.syncId) return
+      this.setState({ isValid })
+    } catch (err) {
+      logger.warn(`Can't check if recipient is valid`, err)
```

Comment:
please undo this because in this case we want the promise to crash so it gets reported to Sentry / display a crash screen / etc..

Reply:
basically we should never try catch without reporting, our approach so far is "let it crash" because if we want to do defensive programming, well, we'll have to do this *everywhere* )

Reply:
( and react error boundaries can help when needed )

Reply:
as it is done in a callback, initiated by didmount/didupdate, it does not bubble up to crash screen. without warning, it's just a plain ugly "uncaught promise fail" in console.

what dyou think we should do in that case?

Reply:
ok for removing the try/catch
nb: it will fail uncaught on invalid addresses

Reply:
the unhandled case is handled by Sentry so yeah it's still better i think